environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/azure/AzureResourceEncryptionParameters.java

```diff
@@ -17,6 +17,8 @@
                     "keyVersion can only contain alphanumeric characters.")
     private String encryptionKeyUrl;
 
+    private String diskEncryptionSetId;
```

Comment:
Please add `@ApiModelProperty` annotation with an appropriate description constant as well.

Reply:
Ping, I cannot see this change. 😃